cli/src/cli.mjs

```diff
@@ -413,9 +414,11 @@ if ( parsed.help ) {
     }
     );
   }
-  else if ( record === 'stop' ) {
-    msg(`Stopping session...`);
-    axios.post(url(host, port, '/api/v1/session/stop'), undefined)
+
+  if ( parsed.sessionOutput ) {
+    const sessionOutput = parsed.sessionOutput;
+    msg(`Set session output to "${sessionOutput}"`);
+    await axios.post(url(host, port, `/api/v1/sessionoutput/${sessionOutput}`), undefined)
     .then(function (response) {
```

Comment:
The combination of await and .then seems unusual. Especially since all we're doing is logging, can we remove the .then?